pkg/startup/startup.go

```diff
@@ -1365,3 +1370,54 @@ func extractKubeadmCIDRs(kubeadmConfig *v1.ConfigMap) (string, string, error) {
 
 	return v4, v6, err
 }
+
+// Check if the string is cidr
+// Since: internal/bytealg/indexbyte_generic.go
+func indexByteString(s string, c byte) int {
+	for i := 0; i < len(s); i++ {
+		if s[i] == c {
+			return i
+		}
+	}
+	return -1
+}
+
+// return the cidr of the dest on local
+func getLocalCidr(ip string, version int) (string, error) {
```

Comment:
nit: CIDR should be same-case in golang, since it is an acronym.

So, this should be `getLocalCIDR`